### DISCUSSION TITLE
SunbirdED portal Installation demo on Windows Machine

### DISCUSSION THREAD

**Discussion by rajeevsathish** [Sunbird-Ed/Community#463] [General Discussion]
Hi Folks!

The Sunbird community will be conducting a virtual demo session on how to install SunbirdED portal locally in a windows machine.
This was one among the most popular queries we received from the community. 

In this session we will demo the installation of SunbirdED on a windows machine using the steps detailed in the [discussion thread](https://github.com/orgs/Sunbird-Ed/discussions/433). We will also answer any queries or resolve any issues that you might have faced during the local installation of SunbirdED on a windows machine. Please leverage this session to get your local instance running in-case you are facing any issues.

Feel free to pass on this info to anybody else who might be interested.

Date : 19th April '23
Time : 5 - 6pm IST

We will be sharing the zoom link soon!

@libin120, @darpandeb, @sudarshanswati, @manthanpandey23, @manish050994, @shwetha-rao13, @techframewirk, @bbmukherjee, @vbg, @dheeraj-nd, @snehar-nd, @vishwab1, @JagadishPujari, @shashankpace, @vinayakmetagoli, @sumanvpacewisdom, @Vivek-M-08, @shreyasb22, @5Amogh, @Ashwiniev95, @nevil-mathew, @Shakthieshwari, @Vinod-V3, @adithyadinesh0412, @suyash2000, @pathiktarento1089 , @BharathTarento, @mahesh-maney, @sajeshkayyath, @santhosh-tarento, @shishirsuman092, @vky25, @karthik-tarento, @SonaArun, @mansurskTarento, @vishalisakar, @radheshhathwarbh, @Xitija, @Hellstellar, @AmithRangadhol04, @krishnakr6, @LavanyaKR3007, @panduankani, @Sanjay-krishnan, @israr-trigyn, @Jyotiannoor, @126251863, @yogeshsarmalkar-trigyn, @ranvsin2, @Debayanpaul1593, @ytk1389 @ravishtrigyn

cc : @madhucr, @Krishnaj20, @vinukumar-vs, @vrayulu, @chitranshu-keshav, @Swayangjit, @Santhosh Gandham, @ashokreddy1208, @PoojaJManjooran, @rajeshkumaravel, @princegupta1131, @abhishekpnt

**Comment by rajeevsathish** [comment]
Hi All,
here is the zoom link details for this session. Look forward to seeing all in the session

Topic: SunbirdED portal Installation demo on Windows Machine
Time: Apr 19, 2023 05:00 PM India
Join Zoom Meeting
https://ekstep-org.zoom.us/j/81731890865?pwd=Z1RsZ0JaZ3JNaDV0SEFiSTVUK25VZz09
Meeting ID: 817 3189 0865
Passcode: 822037

@libin120, @darpandeb, @sudarshanswati, @manthanpandey23, @manish050994, @shwetha-rao13, @techframewirk, @bbmukherjee, @vbg, @dheeraj-nd, @snehar-nd, @vishwab1, @JagadishPujari, @shashankpace, @vinayakmetagoli, @sumanvpacewisdom, @Vivek-M-08, @shreyasb22, @5Amogh, @Ashwiniev95, @nevil-mathew, @Shakthieshwari, @Vinod-V3, @adithyadinesh0412, @suyash2000, @pathiktarento1089 , @BharathTarento, @mahesh-maney, @sajeshkayyath, @santhosh-tarento, @shishirsuman092, @vky25, @karthik-tarento, @SonaArun, @mansurskTarento, @vishalisakar, @radheshhathwarbh, @Xitija, @Hellstellar, @AmithRangadhol04, @krishnakr6, @LavanyaKR3007, @panduankani, @Sanjay-krishnan, @israr-trigyn, @Jyotiannoor, @126251863, @yogeshsarmalkar-trigyn, @ranvsin2, @Debayanpaul1593, @ytk1389 @ravishtrigyn

cc : @madhucr, @Krishnaj20, @vinukumar-vs, @vrayulu, @chitranshu-keshav, @Swayangjit, @Santhosh Gandham, @ashokreddy1208, @PoojaJManjooran, @rajeshkumaravel, @princegupta1131, @abhishekpnt

**Comment by chitranshu-keshav** [comment]
Hi All, 

You can access the recording of this session here -> [Link](https://drive.google.com/file/d/1u_xZstF5vY4SEETqRG8HoGNR1mRvxJow/view?usp=sharing) 

@libin120, @darpandeb, @sudarshanswati, @manthanpandey23, @manish050994, @shwetha-rao13, @techframewirk, @bbmukherjee, @vbg, @dheeraj-nd, @snehar-nd, @vishwab1, @JagadishPujari, @shashankpace, @vinayakmetagoli, @sumanvpacewisdom, @Vivek-M-08, @shreyasb22, @5Amogh, @Ashwiniev95, @nevil-mathew, @Shakthieshwari, @Vinod-V3, @adithyadinesh0412, @suyash2000, @pathiktarento1089 , @BharathTarento, @mahesh-maney, @sajeshkayyath, @santhosh-tarento, @shishirsuman092, @vky25, @karthik-tarento, @SonaArun, @mansurskTarento, @vishalisakar, @radheshhathwarbh, @Xitija, @Hellstellar, @AmithRangadhol04, @krishnakr6, @LavanyaKR3007, @panduankani, @Sanjay-krishnan, @israr-trigyn, @Jyotiannoor, @126251863, @yogeshsarmalkar-trigyn, @ranvsin2, @Debayanpaul1593, @ytk1389 @ravishtrigyn

cc : @madhucr, @Krishnaj20, @vinukumar-vs, @vrayulu, @chitranshu-keshav, @Swayangjit, @Santhosh Gandham, @ashokreddy1208, @PoojaJManjooran, @rajeshkumaravel, @princegupta1131, @abhishekpnt
